docs/faq/items/python2_support.md

```diff
@@ -0,0 +1,6 @@
+# Does Black support Python 2?
+
+For formatting, yes! [Install](../../getting_started.md#installation) with the `python2`
+extra to format Python 2 files too! There are no current plans to drop support, but most
+likely it is bound to happen. Sometime. Eventually. In terms of running _Black_ though,
+Python 3.6 or newer is required.
```

Comment:
Seeing and thinking about this entry again, I wonder whether it would be a good idea to document our Python support policy in general. It's not confirmed but IIRC supported runtime versions are the ones actively supported by CPython devs (i.e. not EOLed). 

Now this can be resolved later once we (ever) hash out our support policy but I thought to mention it.

Reply:
I think the move could be to leave this as it is for now. Once the policy is agreed upon we could add a longer section elsewhere in the documentation, shorten this a bit and put a link between.